routes/web.php

```diff
@@ -584,6 +584,14 @@ function (): void {
                         ->middleware('can:view,jobPoster')
                         ->where('jobPoster', '[0-9]+')
                         ->name('hr_advisor.jobs.preview');
+
+                    Route::get(
+                        'jobs/{jobPoster}/assessment-plan',
+                        'AssessmentPlanController@show'
+                    )
+                        ->middleware('can:manage,jobPoster')
```

Comment:
We already have the view-assessment-plan Gate, we should use that here instead of manage.